lib/xcpretty/formatters/simple.rb

```diff
@@ -59,6 +59,60 @@ def format_compile_asset_catalog(file_path)
       format("Compiling asset catalog", file_path)
     end
 
+    def format_create_build_directory(directory_path, target)
+      format(
+        "Creating build directory",
+        "#{directory_path} in target #{target}"
+      )
+    end
+
+    def format_mkdir(directory_name, target)
+      format("Mkdir", "#{directory_name} in target #{target}")
+    end
+
+    def format_process_product_packaging(entitlements, app_name)
+      format(
+        "Processing product packaging",
+        "#{entitlements} for app #{app_name}"
+      )
+    end
+
+    def format_ditto(source_file, target_file)
+      format("Ditto", "#{source_file} #{target_file}")
```

Comment:
Do we want full paths here (the inputs are paths and not names)? Same question everywhere full path is printed.

Reply:
I followed the rule of thumb I mentioned above.